src/qlever/Qleverfiles/Qleverfile.wikidata

```diff
@@ -1,33 +1,46 @@
-# Qleverfile for Wikidata, use with qlever script (`pip install qlever`)
+# Qleverfile for Wikidata, use with the QLever CLI (`pip install qlever`)
 #
-# qlever get-data  # downloads two .bz2 files of total size ~100 GB
-# qlever index     # takes ~4.5 hours and ~20 GB RAM (on an AMD Ryzen 9 5900X)
-# qlever start     # starts the server (takes a few seconds)
+# qlever get-data  # ~7 hours, ~110 GB (compressed), ~20 billion triples
+# qlever index     # ~5 hours, ~20 GB RAM, ~500 GB (index size on disk)
+# qlever start     # a few seconds, adjust MEMORY_FOR_QUERIES as needed
+#
+# Adding a text index takes an additional ~2 hours and ~50 GB of disk space.
+#
+# Measured on an AMD Ryzen 9 5950X with 128 GB RAM, and NVMe SSD (18.10.2024).
 
 [DEFAULT]
 NAME = wikidata
 
 [data]
-GET_DATA_URL = https://dumps.wikimedia.org/wikidatawiki/entities
-GET_DATA_CMD = curl -LRC - --remote-name-all ${GET_DATA_URL}/latest-all.ttl.bz2 ${GET_DATA_URL}/latest-lexemes.ttl.bz2 2>&1
-VERSION      = $$(date -r latest-all.ttl.bz2 +%d.%m.%Y || echo "NO_DATE")
-DESCRIPTION  = Full Wikidata dump from ${GET_DATA_URL} (latest-all.ttl.bz2 and latest-lexemes.ttl.bz2, version ${VERSION})
+GET_DATA_URL      = https://dumps.wikimedia.org/wikidatawiki/entities
+GET_DATA_CMD      = curl -LROC - ${GET_DATA_URL}/latest-all.ttl.bz2 ${GET_DATA_URL}/latest-lexemes.ttl.bz2 2>&1 | tee wikidata.download-log.txt && curl -sL ${GET_DATA_URL}/dcatap.rdf | docker run -i --rm -v $$(pwd):/data stain/jena riot --syntax=RDF/XML --output=NT /dev/stdin > dcatap.nt
+DATE_WIKIDATA     = $$(date -r latest-all.ttl.bz2 +%d.%m.%Y || echo "NO_DATE")
+DATE_WIKIPEDIA    = $$(date -r wikipedia-abstracts.nt +%d.%m.%Y || echo "NO_DATE")
+DESCRIPTION       = Full Wikidata dump from ${GET_DATA_URL} (latest-all.ttl.bz2 and latest-lexemes.ttl.bz2, version ${DATE_WIKIDATA}) + English Wikipeda abstracts (version ${DATE_WIKIPEDIA}, available via schema:description)
+TEXT_DESCRIPTION  = All English and German literals + all sentences from the English Wikipedia (version ${DATE_WIKIPEDIA}), use with FILTER KEYWORDS(...)
 
 [index]
-INPUT_FILES     = latest-all.ttl.bz2 latest-lexemes.ttl.bz2
-CAT_INPUT_FILES = lbzcat -n 4 -f ${INPUT_FILES}
-SETTINGS_JSON   = { "languages-internal": [], "prefixes-external": [""], "locale": { "language": "en", "country": "US", "ignore-punctuation": true }, "ascii-prefixes-only": true, "num-triples-per-batch": 5000000 }
-STXXL_MEMORY    = 10G
+INPUT_FILES      = latest-all.ttl.bz2 latest-lexemes.ttl.bz2 wikipedia-abstracts.nt dcatap.nt
```

Comment:
This is still required to ensure that all files exist before starting indexing.

Reply:
Is this a question or just a comment? If the latter: I agree

Reply:
Yes that was more of a question. I was ensure for what `INPUT_FILES` is used when `MULTI_INPUT_JSON` is used.